src/version_map.jl

```diff
@@ -36,7 +36,7 @@ const STDLIBS_BY_VERSION = [
             "Distributed",
             UUID("8ba89e20-285c-5b6f-9357-94700520ee1b"),
             nothing,
-            UUID[UUID("9e88b42a-f829-5b0c-bbe9-9e923198166b"), UUID("37e2e46d-f89d-539d-b4ee-838fcccc9c8e"), UUID("9a3f8284-a2c9-5f02-9a11-845980a1fd5c"), UUID("6462fe0b-24de-5631-8697-dd941f90decc")],
+            UUID[UUID("6462fe0b-24de-5631-8697-dd941f90decc"), UUID("9e88b42a-f829-5b0c-bbe9-9e923198166b"), UUID("37e2e46d-f89d-539d-b4ee-838fcccc9c8e"), UUID("9a3f8284-a2c9-5f02-9a11-845980a1fd5c")],
```

Comment:
Sounds like these vectors should be sorted deterministically.  These diffs are nonsense.